src/index.html

```diff
@@ -3,11 +3,393 @@
+            <a href="#aboutUs" class="header__item">about us</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#gallery" class="header__item">gallery</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#consultation" class="header__item">consultation</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#testimonials" class="header__item">testimonials</a>
+          </label>
+          <label for="menuOpener" class="menu__link-closer">
+            <a href="#contacts" class="header__item">contacts</a>
+          </label>
+        </div>
+        <div class="menu__small-link-container">
+          <a href="tel:+12345555555" class="header__item header__item--menu-small-link">
+            +1 234 555-55-55
+          </a>
+          <a href="#" class="header__item header__item--menu-small-link header__item--small-underlined">
+            call to order
+          </a>
+        </div>
+      </section>
+      <!-- middle -->
+    </header>
+
+    <main class="content-container main-tag">
+      <article class="about-us" id="aboutUs">
+        <h2 class="about-us__top-title">
+          Miami best<br>Real estate <span class="bright-emphasis">here</span>
+        </h2>
+        <div class="about-us__text-box">
+          <h2 class="about-us__middle-title bright-emphasis">
+            Over 50 years experience in love.
+          </h2>
+          <p class="about-us__text">
+            Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical. <br><br>
+            The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.
+            <br>Working directly with developers - so have more useful information about the profitable deals.
+            <br><br>
+            <big>We will never be late!<br>Guaranteed secure transaction - 100%</big>
+          </p>
+        </div>
+      </article>
+
+      <article class="gallery" id="gallery">
+        <h2 class="title-with-crown">
+          Gallery
+        </h2>
+        <section class="gallery__object">
+          <div class="gallery__images-row gallery__images-row--2-1">
+            <img
+              src="./images/photos/1.jpg"
+              alt="Marina Palms photo 1"
+              class="gallery__photo"
+            >
+            <img
+              src="./images/photos/2.jpg"
+              alt="Marina Palms photo 2"
+              class="gallery__photo"
+            >
+          </div>
+          <p class="gallery__object-descr">
+            <big class="bright-emphasis">Marina Palms </big>/ North Miami Beach, FL 33162
```

Comment:
have you read about `big`?
it's not supported in HTML 5

Reply:
yes, I do it thanks for linter. I did it using b. But it sad me: "oh no!"

Reply:
span is ok